admin/src/app.jsx

```diff
@@ -244,7 +244,7 @@ const Home = (props) => {
               {/* Institution */}
               <RestrictedRoute path="/mon-etablissement" component={Institution} />
```

Comment:
Vu que l'on met tout en français il faut changer `Institution`
```suggestion
              <RestrictedRoute path="/mon-etablissement" component={Etablissement} />
```